packages/inter-protocol/src/vaultFactory/storeUtils.js

```diff
@@ -5,10 +5,7 @@
 
 // XXX importing these that are declared to be used only for testing
 // until @agoric/store supports composite keys
```

Comment:
This comment no longer applies. It appears these functions are no longer declared to be used only for testing.
https://github.com/endojs/endo/blob/master/packages/marshal/src/encodePassable.js#L385-L389

Reply:
Comment deleted.